### DISCUSSION TITLE
Does it takes time for Galaxy Brain Badge to show up on profile?

### DISCUSSION THREAD

**Discussion by mgks** [drknzz/GitHub-Achievements#15] [Q&A] [answered]
Not sure if there's some condition to get the badge even after getting marked as answer or it takes some time to show up.

**Comment by mgks** [comment, marked as answer]
Ah! Just found out, it needs two verified answers.

**Reply by drknzz** [reply]
Yep, it's instant after you get 2 answers marked.
